go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -364,13 +364,13 @@ func (oc *Controller) createEgressFirewallRules(priority int, match, action, ext
 
 	egressFirewallACL := BuildACL(
 		aclName,
-		nbdb.ACLDirectionToLport,
 		priority,
 		match,
 		action,
 		aclLogging,
+		// since egressFirewall has direction to-lport, set type to ingress
```

Comment:
Is the intuitive interpretation valid as well? As in, this would be an ACL that would apply on ingress to the switch?
Is more intuitiveness the reason you introduced the new type?

Reply:
egress firewall will be updated to use egressACLType later, since it should be in the same pipeline with other egress acls, like egress network policy, but that should be a part of egress firewall refactoring
here I added a comment to show using ingressACLType is intentional for egress firewall, which may not be obvious

Reply:
```
       direction: string, either from-lport or to-lport
              Direction of the traffic to which this rule should apply:

              ·      from-lport: Used to implement filters on traffic arriving
                     from a logical port. These rules are applied to the logi‐
                     cal switch’s ingress pipeline.

              ·      to-lport:  Used to implement filters on traffic forwarded
                     to a logical port. These rules are applied to the logical
                     switch’s egress pipeline.
```
This is counter-intuitive so I am not sure we are picking the best names for aclType values.

It looks to me like it should be more `lportIngress` and `lportEgress` with at least depicts clearly in relation to what ingress or egress refers to.